package.json

```diff
@@ -13,7 +16,8 @@
   },
   "devDependencies": {
     "concurrently": "^3.5.0",
-    "husky": "^0.13.3"
+    "husky": "^0.13.3",
+    "lerna": "^3.10.7"
```

Comment:
lerna intended?

Reply:
I mentioned this in person - but lerna is added here so we use the version command to update the version numbers in all the package.json files in each workspace.

https://github.com/lerna/lerna/tree/master/commands/version